app/src/main/java/de/mytfg/app/android/MyTFG.java

```diff
@@ -3,6 +3,21 @@
 import android.app.Application;
 import android.content.Context;
 
+import org.acra.*;
+import org.acra.annotation.*;
+import org.acra.sender.HttpSender;
+
+@ReportsCrashes(
+        httpMethod = HttpSender.Method.PUT,
+        reportType = HttpSender.Type.JSON,
+        formUri = "https://acra.mytfg.de/acra-mytfg_app_android/_design/acra-storage/_update/report",
```

Comment:
ACRA sendet im Fall einer Exception der App einen Absturzbericht (Stacktrace, Device Infos, ...).
Der Bericht wird als JSON an die in firmUri genannte Adresse per HTTP(s) PUT gesendet.
Wir können die Berichte dann [hier](https://acra.mytfg.de/acralyzer/_design/acralyzer/index.html) ansehen.

Der User bekommt dann ein Toast mit dem String R.string.crash_toast (*Leider ist die MyTFG App abgestürzt, ein Absturzbericht wird gesendet*) zu sehen.